styles/styles.css

```diff
@@ -0,0 +1,22 @@
+p {
+  text-align: center;
+  width: 306px;
+  height: 47px;
+  margin: 120px 567px 20px 567px;
```

Comment:
Try to delete the rows 3-5 and you will understand that they have no sense.

Reply:
Ok, I deleted